packages/landing/src/config.map.ts

```diff
@@ -162,6 +166,13 @@ export class MapConfig extends Emitter<MapConfigEvents> {
     this.emit('change');
   }
 
+  setFilterDateRange(after: string | undefined, before: string | undefined): void {
+    this.filter.date.after = after;
```

Comment:
this needs to check to see if it actually changed, before actually firing the events.